dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/organisationunit/OrganisationUnitGroupStoreTest.java

```diff
@@ -58,4 +63,20 @@ public void testGetOrganisationUnitGroupsWithGroupSets()
 
         assertContainsOnly( groupStore.getOrganisationUnitGroupsWithGroupSets(), withSet );
     }
+
+    @Test
+    public void testGetOrgUnitGroupInGroupSet()
+    {
+        OrganisationUnit organisationUnitA = addOrganisationUnit( 'A' );
+        OrganisationUnit organisationUnitB = addOrganisationUnit( 'B' );
+        OrganisationUnit organisationUnitC = addOrganisationUnit( 'C' );
+        OrganisationUnitGroup groupA = addOrganisationUnitGroup( 'A', organisationUnitA );
+        OrganisationUnitGroup groupB = addOrganisationUnitGroup( 'B', organisationUnitB );
+        OrganisationUnitGroup groupC = addOrganisationUnitGroup( 'C', organisationUnitC );
+        Set<OrganisationUnitGroup> groups = Sets.newHashSet( groupA, groupB, groupC );
+
+        OrganisationUnitGroupSet groupSet = addOrganisationUnitGroupSet( 'A', groupA );
```

Comment:
Would it be better to include a all the groups in the group set to verify that the method returns the correct group out of many?

Reply:
fixed